src/cmds/qsub.c

```diff
@@ -4830,6 +4830,15 @@ main(int argc, char **argv, char **envp)   /* qsub */
 	if (command_flag == 0) {
 		/* if script is empty, get standard input */
 		if ((strcmp(script, "") == 0) || (strcmp(script, "-") == 0)) {
+			/* if this is a terminal, print a short info */
+			if (isatty(STDIN_FILENO) && Interact_opt == FALSE) {
```

Comment:
I did a little bit of internet sleuthing and it looks like isattr() is deprecated on windows(https://msdn.microsoft.com/en-us/library/ms235388.aspx).  I don't know if this means we should use it or not.  It's still a perfectly valid function in the UNIX world (http://pubs.opengroup.org/onlinepubs/9699919799/).  Should we use isatty() anyway or put it in an ifdef block and use _isatty() on windows and isatty() on linux?

Reply:
@bhroam We are using _isatty() in windows... see this https://github.com/PBSPro/pbspro/blob/master/src/include/win.h#L70

Reply:
Thanks for pointing that out @hirenvadalia.  I do not see us including win.h in qsub.  Maybe its needed?

Reply:
@bhroam It is getting included via windows version of pbs_config.h (see this https://github.com/PBSPro/pbspro/blob/master/win_configure/include/pbs_config.h#L83)